keras/layers/convolutional_recurrent.py

```diff
@@ -331,6 +331,8 @@ def __init__(self, filters,
 
         self.dropout = min(1., max(0., dropout))
         self.recurrent_dropout = min(1., max(0., recurrent_dropout))
+        if 0 < (self.dropout + self.recurrent_dropout):
```

Comment:
I agree, but I followed the old code, I mean the code from Recurrent class (in the call function if I remember correctly)
but if you say so, I'll change it